### PR TITLE
DTSPO-6258 - dev: add rbac to vh namespace

### PR DESCRIPTION
**Pull request by endakelly**
- add image to jd
- Add role to vh

### CODE REVIEW

File: apps/vh/base/nonprod-role.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+metadata:
+  name: nonprod-team-permissions
+rules:
+- apiGroups: ['']
+  resources: ['pods']
+  verbs: ['delete']
+- apiGroups: ['']
+  resources: ['secrets']
+  verbs: ['*']
+- apiGroups: ['helm.fluxcd.io']
```
Comment (timja) [inline apps/vh/base/nonprod-role.yaml:12]: don't we need a new api group for flux 2?

ref https://hmcts.github.io/ways-of-working/troubleshooting/#latest-image-is-not-updated-in-cluster

File: apps/vh/base/nonprod-role.yaml [hunk 2]
```diff
@@ -12,6 +12,9 @@ rules:
 - apiGroups: ['helm.fluxcd.io']
   resources: ['helmreleases']
   verbs: ['delete']
+- apiGroups: ['v2beta1.helm.fluxcd.io']
```
Comment (timja) [inline apps/vh/base/nonprod-role.yaml:15]: this definitely right and not?
```suggestion
- apiGroups: ['helm.fluxcd.io']
```

Comment (endakelly) [reply apps/vh/base/nonprod-role.yaml:15]: I was looking at the output of `k get apiservices.apiregistration.k8s.io -A`. There's v1.helm.fluxcd.io and v2beta1.helm.fluxcd.io. Will helm.fluxcd.io do it for both?

Comment (timja) [reply apps/vh/base/nonprod-role.yaml:15]: I think so

Comment (endakelly) [reply apps/vh/base/nonprod-role.yaml]: I've reverted it there

**Review by timja — APPROVED** [review]

**Pull request merged**